### PR TITLE
Update changelog for v2.4.0

### PR DESCRIPTION
**Pull request by demartinofra**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,32 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
 - Add support for Ubuntu in China region `cn-northwest-1`
 
+**CHANGES**
+- SGE: changed following parameters in global configuration
+  - `max_unheard 00:03:00`: allows a faster reaction in case of faulty nodes
+  - `reschedule_unknown 00:00:30`: enables rescheduling of jobs running on failing nodes
+  - `qmaster_params ENABLE_FORCED_QDEL_IF_UNKNOWN`: forces job deletion on unresponsive nodes
+  - `qmaster_params ENABLE_RESCHEDULE_KILL`: forces rescheduling or killing of jobs running on failing nodes
+- Slurm: decrease SlurmdTimeout to 120 seconds to speed up replacement of faulty nodes
+- Always use full master FQDN when mouing nfs on compute nodes. This solves some issues occurring with some networking
```
Comment (enrico-usai) [inline CHANGELOG.md]: typo: mounting

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,9 +7,32 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 -----
 
 **ENHANCEMENTS**
-
+- Add support for EFA on Centos 7, alinux and Ubuntu 1604
```
Comment (enrico-usai) [inline CHANGELOG.md]: alinux --> Amazon Linux ? --> to be aligned with others OS names

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,9 +7,32 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
+- Removed unused `max_queue_size` from sqswatcher.cfg
+- Remove double quoting of the post_install args
+
+
+**BUG FIXES**
+- Fix issue that was preventing Torque from being used on Centos 7
+- Start node daemons at the end of instance initialization. The time spent for post-install script and node
+  initialization is not counted as part of node idletime anymore.
+- Fix issue which was causing an additional and invalid EBS mount point to be added in case of multiple EBS
+- Install Slurm libpmpi/libpmpi2 that is distributed in a separate package since Slurm 17
```
Comment (enrico-usai) [inline CHANGELOG.md]: since Slurm 17 or 18?

File: CHANGELOG.md (hunk 1 above)
Comment (demartinofra) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (demartinofra) [reply CHANGELOG.md]: 👍

File: CHANGELOG.md (hunk 3 above)
Comment (demartinofra) [reply CHANGELOG.md]: we moved from 16 to 17 but that changed in version 17

**Pull request merged**